### PR TITLE
Add GM Closure Notice component to Help Contact

### PR DESCRIPTION
**Pull request by mirka**
#### Changes proposed in this Pull Request

* Adds a reusable GMClosureNotice component to notify users about Automattic's support level over the Grand Meetup.
* Sets up a notice for the 2019 Grand Meetup on the Contact form (both the dedicated page and the Inline Help widget).
* _Clean up:_ Removes the Live Chat Closure Notice for Easter 2019.

Screen Shot 2019-08-29 at 23 40 41

#### Testing instructions

1. Open the Inline Help widget or go to `/help/contact`.
2. Alter your system clock to check messages for times before/after the `displayAt`, `defaultCloses/ReopensAt`, and `businessAndEcommerceCloses/ReopensAt` times.

Fixes 663-gh-hg

### CODE REVIEW

File: client/me/help/gm-closure-notice/index.jsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+	const mainMessageArgs = {
+		closesAt: closesAt.format( DATE_FORMAT_LONG ),
+		closedUntil: closedUntil.format( DATE_FORMAT_LONG ),
+		reopensAt: reopensAt.format( DATE_FORMAT_LONG ),
+	};
+
+	const MAIN_MESSAGES = {
+		before: {
+			hasPlan: translate(
+				'Live chat support will be closed %(closesAt)s through %(closedUntil)s, included. Email support will be open during that time, and we will reopen live chat on %(reopensAt)s.',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Live chat support will be closed from %(closed_start_date)s through %(closed_end_date)s, included. Email support will be open during that time, and we will reopen live chat on %(support_resume_date)s.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/gm-closure-notice/index.jsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+				{ args: mainMessageArgs }
+			),
+			nonPlan: translate(
+				'Private support will be closed %(closesAt)s through %(closedUntil)s, included. We will reopen private support on %(reopensAt)s.',
+				{ args: mainMessageArgs }
+			),
+		},
+		during: {
+			hasPlan: translate(
+				'Live chat support will be closed through %(closedUntil)s, included. Email support will be open during that time, and we will reopen live chat on %(reopensAt)s.',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Live chat support will be closed for the Thanksgiving holiday on %(closed_start_date)s. Email support will be open during that time, and we will reopen live chat on %(closed_end_date)s.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/gm-closure-notice/index.jsx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+				{ args: mainMessageArgs }
+			),
+			nonPlan: translate(
+				'Private support will be closed through %(closedUntil)s, included. We will reopen private support on %(reopensAt)s.',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Private support will be closed through %(closed_end_date)s, included. We will reopen private support on %(support_resume_date)s.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/gm-closure-notice/index.jsx [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+				{ args: mainMessageArgs }
+			),
+		},
+	};
+
+	const REASON_MESSAGES = {
+		hasPlan: translate(
+			'Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together to work on improving our services, building new features, and learning how to better serve our customers like you. But never fear! If you need help in the meantime, you can submit an email ticket through the contact form: {{contactLink}}https://wordpress.com/help/contact{{/contactLink}}',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '{{p}}Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together to work on improving our services, building new features, and learning how to better serve you, our users. But never fear! If you need help in the meantime, you can submit an email ticket through the contact form.{{/p}}' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '{{p}}Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together to work on improving our services, building new features, and learning how to better serve you, our users. But never fear! If you need help in the meantime, you can submit an email ticket through the contact form.{{/p}}' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/gm-closure-notice/index.jsx [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+			{
+				components: {
+					supportLink: <a href="https://en.support.wordpress.com" />,
+				},
+			}
+		),
+	};
+
+	const FORUMS_NOTE = translate(
+		'Our staff will be keeping an eye on the {{forumLink}}Forums{{/forumLink}} for urgent matters.',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Our staff will be keeping an eye on the {{link}}Forums{{/link}} for urgent matters.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/gm-closure-notice/index.jsx [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+			<FoldableCard
+				className="gm-closure-notice"
+				clickableHeader={ true }
+				compact={ true }
+				header={ HEADING }
+			>
+				{ mainMessage }
+				<br />
+				<br />
+				{ translate( '{{contactLink}}Read more{{/contactLink}}', {
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( '{{contactLink}}Read more.{{/contactLink}}' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/gm-closure-notice/index.jsx [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+				{ args: mainMessageArgs }
+			),
+			nonPlan: translate(
+				'Private support will be closed from %(closed_start_date)s through %(closed_end_date)s, included. We will reopen private support on %(support_resume_date)s.',
+				{ args: mainMessageArgs }
+			),
+		},
+		during: {
+			hasPlan: translate(
+				'Live chat support will be closed through %(closed_end_date)s, included. Email support will be open during that time, and we will reopen live chat on %(support_resume_date)s.',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Live chat support will be closed for the Thanksgiving holiday on %(closed_start_date)s. Email support will be open during that time, and we will reopen live chat on %(closed_end_date)s.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/gm-closure-notice/index.jsx [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
+	if ( ! currentDate.isBetween( displayAt, reopensAt ) ) {
+		return null;
+	}
+
+	const HEADING = translate( 'Limited Support %(closesAt)s – %(reopensAt)s', {
+		args: {
+			closesAt: closesAt.format( DATE_FORMAT_SHORT ),
+			reopensAt: reopensAt.format(
+				reopensAt.isSame( closesAt, 'month' ) ? 'D' : DATE_FORMAT_SHORT
```
Comment (mattwondra) [inline client/me/help/gm-closure-notice/index.jsx]: Nice touch 💯

**Pull request merged**